gabrielaEnache/src/main/java/com/endava/generics/GenericLibrary.java

```diff
@@ -0,0 +1,62 @@
+package com.endava.generics;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class GenericLibrary<T extends Media> {
+    private List<T> items;
+
+    public GenericLibrary() {
+        items = new ArrayList<>();
+    }
+
+    public void addItem(T item) {
+        if (items.size() < 6)
```

Comment:
6, magic number, please extract as a constant and assign a meaningful name

Reply:
Done